fdbclient/NativeAPI.actor.cpp

```diff
@@ -7021,6 +7027,7 @@ ACTOR Future<ProtocolVersion> getClusterProtocolImpl(
 
 	loop {
 		if (!coordinator->get().present()) {
+			TraceEvent("GetClusterProtocolWaitForCoordinator").log();
```

Comment:
The call to `log` isn't necessary, it's implied by the destruction of the object.

```suggestion
			TraceEvent("GetClusterProtocolWaitForCoordinator");
```

Reply:
I would also think it is not necessary, but I found .log() being called everywhere, where event details are not provided. I will try remove it and see what happens.